packages/flutter/lib/src/widgets/form.dart

```diff
@@ -638,4 +684,8 @@ enum AutovalidateMode {
   /// Used to auto-validate [Form] and [FormField] only after each user
   /// interaction.
   onUserInteraction,
+
+  /// In order to validate all fields of a [Form] after the first time the user interacts
+  /// with one, use [always] instead.
```

Comment:
Did you used to have another sentence here that got deleted, or am I misremembering? I think it should start with "Used to..." and the second paragraph should be this sentence.